src/util.ts

```diff
@@ -1139,6 +1139,27 @@ export class Util {
             source: 'brs'
         };
     }
+
+    /**
+     * Finds the array of callables from a container map, taking into account the function from which it was called
+     * If the callable was called in a function in a namespace, functions in that namespace are preferred
+     * @return an array with callable containers - could be empty if nothing was found
+     */
+    public getCallableContainersFromContainerMapByFunctionCall(callablesByLowerName: CallableContainerMap, expCall: FunctionCall): CallableContainer[] {
```

Comment:
This name is crazy! Is there any way we could find a shorter name without lots of abbreviations?

Reply:
No... It;s impossible